mapbox/tilejson/tilejson.go

```diff
@@ -136,3 +141,74 @@ type VectorLayer struct {
 	// Tegola supports individual layer tiles.
 	Tiles []string `json:"tiles"`
 }
+
+//SetVectorLayers fill VectorLayers from map layers
+func (tileJSON *TileJSON) SetVectorLayers(layers []atlas.Layer) {
```

Comment:
I think we should move this to atlas/map.go. The intent of the `mapbox` packages is to keep them independent of the implementation so they could potentially be moved out of the tegola codebase and stand alone. By adding references to the `atlas` package the `tilejson` package is now dependent on the `tegola` codebase.

Reply:
I move the common code to the atlas package. I will also update the PR #655